cli/preprocessor.go

```diff
@@ -95,13 +96,16 @@ func analyze(
 		*compiler = "cpp"
 	}
 	if lang != "" {
-		*lineCommentChars, *blockCommentCharsOpen, *blockCommentCharsClose = getCommentCharsFromLang(lang)
+		*lineCommentIden, *blockCommentIdenOpen, *blockCommentIdenClose = getCommentIdenFromLang(lang, *fileInput)
 	}
 	// Ensure value of comment char
-	if *lineCommentChars == "" && *blockCommentCharsOpen == "" && *blockCommentCharsClose == "" {
-		*lineCommentChars = "#"
-		*blockCommentCharsOpen = ""
-		*blockCommentCharsClose = ""
+	if *lineCommentIden == "" && *blockCommentIdenOpen == "" && *blockCommentIdenClose == "" {
+		*lineCommentIden = "#"
+		*blockCommentIdenOpen = ""
+		*blockCommentIdenClose = ""
```

Comment:
? shouldn't this be an error as well?

Reply:
https://github.com/compose-generator/ccom/blob/86f126e1e83fb945f25e0d54dcf0dbe6076bcaec/compiler/java/src/main/java/com/c/lexer/Lexer.java#L116-L122

I implemented this as an error

Reply:
Yes, you are right. Thats another remnant here ;)
Will fix that quickly ...